integration-tests/utils/catalog-utils.js

```diff
@@ -14,6 +14,7 @@ const eitherThrow = (testFailure, { cleanupFailure, cleanupMessage }) => {
 //chance to dataset with asses
 const linkDataToWorkspace = async (page, testUrl, token, datasetName) => {
   await navigateToDataCatalog(page, testUrl, token)
+  await verifyAccessibility(page)
```

Comment:
Is this something we should be calling in other tests too? I'd guess we'd want to check both the list and the details page.

Reply:
It looks like your changes impacted how the data catalog is enabled in the integration tests.

Reply:
@cahrens interestingly, it passes locally, just not on staging. I am wondering if its because this hasn't deployed to staging. Do you know if we deploy a new UI pointing at staging? Or just run the tests against the staging UI?